config.yaml

```diff
@@ -0,0 +1,42 @@
+options:
+  email:
+    type: string
+    description: Account email address
+  server:
+    type: string
+    description: Certificate authority server
+    default: "https://acme-v02.api.letsencrypt.org/directory"
+  aws_access_key_id:
+    type: string
+    default: ""
+    description: AWS Access Key ID
+  aws_secret_access_key:
+    type: string
+    default: ""
+    description: AWS Secret Access Key
+  aws_region:
+    type: string
+    description: AWS Region
+  aws_shared_credentials_file:
+    type: string
+    default: ""
+    description: Path to the file that contains the AWS credentials
```

Comment:
This does not seem used by the code and I assume the file would need to be uploaded to the container. I think we should get rid of it.

Reply:
✅